### PR TITLE
Milestone 1: Create your own Enumerable

### PR DESCRIPTION
**Pull request by SaidRasinlic**
### In this milestone we've completed the following things:

- Created a class MyList that has an instance variable @list.
- In MyList we implemented a method #each that yields successive members of @list and uses the MyEnumerable module.
- Created a module MyEnumerable that implements the following methods (they have the same funcionality as methods in Enumerable (_#all?, #any?, #filter_).
- Each class and module have a separate .rb file.
- Successfully verified our solutions and it works perfectly.

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&)
+    @list.each(&)
+  end
```
Comment (julie-ify) [inline my_list.rb:12]: - [ ] I am unable to run the tests due to an error in this method. Kindly fix it.

  raid-1

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @SaidRasinlic and @aboongm,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: my_list.rb (hunk 1 above)
Comment (SaidRasinlic) [reply my_list.rb:12]: ## @julie-ify

**We've just tested the application on both of our PC-s (Windows systems) and it works perfectly. Here is a screenshot for the proof.**

Ruby-Enumerable
.

**Review by brytebee — APPROVED** [review]
Hi, @SaidRasinlic👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**